templates/libvirt/xen.xml

```diff
@@ -2,11 +2,12 @@
     {% block basic %}
         <name>{{ vm.name }}</name>
         <uuid>{{ vm.uuid }}</uuid>
-        {% if vm.virt_mode == 'hvm' and vm.devices['pci'].persistent() | list %}
+        {% if ((vm.virt_mode == 'hvm' and vm.devices['pci'].persistent() | list)
+            or vm.maxmem == 0) -%}
```

Comment:
Here: this should be `vm.is_memory_balancing_possible()`.

Reply:
I don't think it should be `vm.is_memory_balancing_possible()`. If anything, it should be just `vm.maxmem==0` (and let the core handle the rest).

Reply:
Hmm, you're right. If balancingis not possible and maxmem != 0, the XML will not be rendered at all.